src/mpl2/src/hier_rtlmp.h

```diff
@@ -271,6 +271,7 @@ class HierRTLMP
 
   // technology-related variables
   float dbu_ = 0.0;
+  int manufacturing_grid_ = 10;  // the manufacture grid in dbu
```

Comment:
If no grid exists the default should be 1 dbu

Reply:
Is 1 dbu the right default?  Also is it OK for tech lefs not to have manufacturing grid

Reply:
fixed

Reply:
LEF allows it to be optional though in practice I never seen it not specified.  If you didn't then any coordinate is presumably legal so 1dbu would be correct.